setup-dev-env.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
do we need to provide a bat file equivalent for windows?

Reply:
Yes, but we'll make that a separate PR.